Sources/Helpers/RemoteConfigHelper.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  File.swift
+//  
+//
+//  Created by Emad Ghorbania on 11/11/2022.
+//
+
+import FirebaseRemoteConfig
+
+public struct RemoteConfigHelper {
```

Comment:
Should we have DocC comments here?

Reply:
Is `RemoteConfigHelper` only intended to be used internally in this library or is it OK that it is exposed to clients of this library?

Reply:
They need to know the type of error that they get, also it's a helper if you want you can use it

Reply:
and then I add docs in next commit